lib/kennel/tasks.rb

```diff
@@ -35,10 +35,12 @@ def ci
         load_environment
 
         if on_default_branch? && git_push?
-          Kennel.strict_imports = false
-          Kennel.update
+          config = Kennel::Config.from_env do |c|
+            c.strict_imports = false
+          end
```

Comment:
another neat api could be:
```suggestion
          config = Kennel::Config.from_env.merge(strict_imports: false)
```